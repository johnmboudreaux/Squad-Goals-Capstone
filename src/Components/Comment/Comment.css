.comment {
  margin: 10px 20px;
  height: 250px;
  width: 95%; }

.comment p {
  color: #282828;
  margin-right: auto;
  text-align: left; }
