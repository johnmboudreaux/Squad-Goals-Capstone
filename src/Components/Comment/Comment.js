import React from 'react';
import PropTypes from 'prop-types';

const Comment = ({ commentData }) => {
  return (
    <div className="smack-talk">
      <p>{commentData.body}</p>
    </div>
  );
};

Comment.propTypes = {
  commentData: PropTypes.array,
};

export default Comment;
