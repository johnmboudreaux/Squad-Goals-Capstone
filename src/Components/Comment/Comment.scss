@import '../../variables.scss';

.comment {
  margin: 10px 20px;
  height: 250px;
  width: 95%;
}

.comment p {
  color: $black;
  margin-right: auto;
  text-align: left;
}
