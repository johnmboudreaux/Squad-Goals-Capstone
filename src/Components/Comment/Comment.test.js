import { shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import Comment from './Comment';
import commentData from '../../../data/test/comments_data_test';

describe('MainApp snapshot', () => {
  it('should always match the snapshot', () => {
    const wrapper = shallow(<Comment
      commentData={commentData}
    />);

    expect(wrapper).toMatchSnapshot();
  });


  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Comment
      commentData={commentData}
    />, div);
  });
});
