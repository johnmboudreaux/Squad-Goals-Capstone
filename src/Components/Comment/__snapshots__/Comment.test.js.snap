// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainApp snapshot should always match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Comment
    commentData={
      Array [
        Object {
          "body": "this is the comments body",
          "conversation_id": 1,
          "id": 1,
        },
        Object {
          "body": "this is the comments body 2",
          "conversation_id": 1,
          "id": 2,
        },
        Object {
          "body": "this is the comments body 3",
          "conversation_id": 3,
          "id": 3,
        },
        Object {
          "body": "this is the comments body 4",
          "conversation_id": 4,
          "id": 4,
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p />,
      "className": "smack-talk",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p />,
        "className": "smack-talk",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
